db/migrate/20170221003240_create_events.rb

```diff
@@ -0,0 +1,11 @@
+class CreateEvents < ActiveRecord::Migration[5.0]
+  def change
+    create_table :events do |t|
+      t.date :date
```

Comment:
Because we never want these to be blank values we should also include a `null: false` for each of them. The Rails validations are great but if we really don't want these to be blank we should have DB validations as well.